tests/configs.spec.ts

```diff
@@ -281,4 +281,103 @@ describe('configs', () => {
       expect(isValid).to.be.false
     })
   })
+
+  describe('ROUTE_CAVEAT_CONFIGS', () => {
+    let config: BoltwallConfig,
+      satisfier: Satisfier,
+      getCaveats: CaveatGetter,
+      condition: string
+
+    beforeEach(() => {
+      config = ROUTE_CAVEAT_CONFIGS
+      condition = 'route'
+
+      if (!config.caveatSatisfiers)
+        throw new Error('Missing caveat satisfiers on route config')
+
+      if (!config.getCaveats)
+        throw new Error('Missing caveat getter from route config')
+
+      satisfier = Array.isArray(config.caveatSatisfiers)
+        ? config.caveatSatisfiers[0]
+        : config.caveatSatisfiers
+
+      getCaveats = Array.isArray(config.getCaveats)
+        ? config.getCaveats[0]
+        : config.getCaveats
+    })
+
+    it('should create a caveat that restricts access by route and be able to satisfy it', () => {
+      const request: any = { path: '/path' }
```

Comment:
not a fan of `any` if it can be avoided. I think it should be ok leaving this not typed and the type will just be inferred from the object which should be fine for the test